### DISCUSSION TITLE
proxy usage

### DISCUSSION THREAD

**Discussion by DevSamuelOliveira** [kevinzg/facebook-scraper#569] [Q&A]
Is there any parameter that can be passed a proxy? or is it easy to implement?

**Comment by neon-ninja** [comment]
Yes - try the `set_proxy` function

**Reply by DevSamuelOliveira** [reply]
I'm sorry for the ignorance but I'm a python layman, could you give a little example.

Would it be something like?

get_profile("profile_id", set_proxy("proxy"))

or 

get_profile("profile_id", set_proxy="proxy")

**Reply by neon-ninja** [reply]
No, you would call the set_proxy function prior to calling get_profile. Like:

```python
from facebook_scraper import *
set_proxy("http://localhost:8080")
print(get_profile("profile_id"))
```
